main.go

```diff
@@ -189,10 +189,11 @@ func wrappedMain() int {
 		var TTY packer.TTY
 		if !inPlugin {
 			var err error
-			TTY, err = tty.Open()
+			TTY, err := tty.Open()
```

Comment:
```suggestion
			TTY, err = tty.Open()
```
The tty created line `189` won't be set if you add a semicolon here

Reply:
I thought about this, but I was not unsure whether to add Close method to  TTY interface declared in `packer/tty.go` or use `tty.Close` from `github.com/mattn/go-tty` library.
Thank you for pointing this.